flixel/addons/effects/FlxGlitchSprite.hx

```diff
@@ -69,17 +86,12 @@ class FlxGlitchSprite extends FlxSprite
 		}
 		else
 		{
-			_time += elapsed;
+			_time += FlxG.elapsed;
```

Comment:
Yes, exactly why I wanted to avoid it... I don't know what else to change, though...